server_action_navigate/__manifest__.py

```diff
@@ -1,17 +1,18 @@
+# Migration 2022 - 14.0: Ashish Hirpara (https://ashish-hirpara.com)
 # Copyright (C) 2020 - Today: GRAP (http://www.grap.coop)
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Server Actions - Navigate",
-    "version": "13.0.1.0.0",
-    "author": "GRAP, " "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Ashish Hirpara, GRAP, Odoo Community Association (OCA)",
     "summary": "Navigate between any items of any Odoo Models",
     "category": "Tools",
     "website": "https://github.com/OCA/server-backend",
     "license": "AGPL-3",
     "depends": ["base"],
-    "maintainers": ["legalsylvain"],
+    "maintainers": ["ashishhirpara"],
```

Comment:
@legalsylvain are we sure about this?

Reply:
@AshishHirapara. This is a list, if you want to maintain this module no problem. (I think it's the first time modules are comaintained, but let's try !)

could you replace by ``"maintainers": ["legalsylvain", "ashishhirpara"]``

regards.

Reply:
Sure, done. thanks.